content/blog/2025/05/02/2025-05-02-pipeline-graph-view.adoc

```diff
@@ -0,0 +1,68 @@
+---
+layout: post
+title: "Revamped Pipeline visualization in Jenkins"
+tags:
+- jenkins
+- user-experience
+- ux
+- pipeline
+authors:
+- janfaracik
+opengraph:
+  image: /images/post-images/2025/05/2025-05-02-pipeline-graph-view/opengraph.png
+---
+
+image::/images/post-images/2025/05/2025-05-02-pipeline-graph-view/opengraph.png[role=center]
+
+We're excited to announce a major update to the https://plugins.jenkins.io/pipeline-graph-view/[Pipeline Graph View]
+plugin - a complete redesign that introduces a modern, user-focused interface that enhances pipeline visualization,
+customization, and performance.
+
+---
+
+=== So what's new?
+
+**Rewritten and redesigned**
+
+The plugin has been completely rebuilt with a clean, modern interface using the latest version of React. We've put a lot
+of effort into simplifying the plugin's structure - switching from Webpack to Rspack, moving to Vite, and replacing 
+class-based components with modern function components. These changes made the codebase easier to maintain and work 
+with, and along the way, we removed thousands of lines of code to boost both maintainability and performance.
+
+The interface, with new components from the link:/blog/2025/01/10/design-library/[Jenkins Design Library], has been
+thoughtfully redesigned to offer a smoother, more intuitive experience. The plugin now supports full-page scrolling -
+eliminating the need for nested scroll areas - along with sticky headings for better context. The pipeline graph can be
+panned and zoomed, allowing users to explore complex pipelines with ease. Subtle, delightful animations enhance
+usability without getting in the way. Stage progress is now clearly visible, and loading skeletons provide immediate
+visual feedback, resulting in a more stable and responsive experience.
+
+**Unified view**
+
+The new interface combines the Pipeline graph, stages, and logs into a single, streamlined layout. You can inspect a
+stage, follow its execution, and read its output - all in one place.
+
+**Customization**
+
+The Pipeline graph appears on top by default, but you can move it to the left to better suit ultrawide displays. Columns
+can be resized and hidden, and layout preferences are saved automatically.
+
+image::/images/post-images/2025/05/2025-05-02-pipeline-graph-view/sidebar.png[role=center]
+
+**Performance improvements**
+
+We've made significant behind-the-scenes improvements to how stages and logs are fetched and rendered. The new plugin
+loads faster, scrolls smoother, and handles larger pipelines more gracefully.
+
+---
+
+=== Available now 🚀
+
+The new Pipeline Graph View is available now for users on Jenkins 2.479.3 and above, head to Plugins to get the update.
```

Comment:
```suggestion
The new Pipeline Graph View is available now for users on Jenkins 2.479.3 and later.
If you haven't installed the plugin already, be sure to download it through the Plugins section of your Jenkins controller to experience the update.
```
Small wording suggestion to be more clear in where the plugin is accessed (only since it could be seen as either plugins.jenkins.io or the plugins section of a jenkins controller).